nni/common/graph_utils.py

```diff
@@ -388,6 +389,7 @@ def _expand_module_node(self, node, node_name, unique_name, op_type, nodes,
                                 visited.add(successor_node)
                         else:
                             outputs.append(output_name)
+                            break
```

Comment:
why add `break` here, there may be other `outputs`/`successor_node` in `nodes`?

Reply:
Actually, this is to remove duplicated output tensors which caused by a legacy problem(use list to save the output name rather than set, and we should use the list indeed because it preserve the order of the input parameters).
I have updated the code and use a better way to remove the duplication in the outputs, thanks.